cinema/views.py

```diff
@@ -56,3 +90,21 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderSetPagination(PageNumberPagination):
+    page_size = 10
+    page_size_query_param = "page_size"
+    max_page_size = 100
```

Comment:
Consider moving the `OrderSetPagination` class to a separate pagination.py file in the cinema app for better code organization and reusability.
This follows the Django best practice of separating different types of functionality into dedicated modules.